content/oci/oci.go

```diff
@@ -54,8 +54,9 @@ type Store struct {
 	indexPath     string
 	index         *ocispec.Index
 	indexLock     sync.Mutex
+	sync          sync.RWMutex
```

Comment:
Request documentation on `sync`. Basically, `sync` defines two regions: one can be executed concurrently (via `RLock()` and `RUnlock()`) and another cannot (via `Lock()` and `Unlock()`)

Reply:
added doc comments.